src/api/app/controllers/webui/package_controller.rb

```diff
@@ -897,4 +922,24 @@ def set_job_status
       @buildtime = nil
     end
   end
+
+  def text_width(text)
+    file = TTFunk::File.open('/usr/share/fonts/truetype/DejaVuSans.ttf')
```

Comment:
Is there a way we can expose this path to be configurable with `Makefile.include`?

In Fedora, this is stored as `/usr/share/fonts/dejavu-sans-fonts/DejaVuSans.ttf`.